<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калькулятор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>62</y>
      <width>561</width>
      <height>272</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Oriya</family>
     </font>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0" colspan="3">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="X_Btn">
         <property name="font">
          <font>
           <family>Noto Sans Oriya</family>
          </font>
         </property>
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Noto Sans Oriya</family>
          </font>
         </property>
         <property name="text">
          <string>=</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="editX">
         <property name="font">
          <font>
           <family>Noto Sans Oriya</family>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="maxLength">
          <number>256</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="ac_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="open_Bkt_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="close_Bkt_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="n7_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="n8_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="n9_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="n4_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="n5_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="n6_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="n1_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="n2_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="n3_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="n0_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="dot_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="7">
      <widget class="QLineEdit" name="editCalc">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="maxLength">
        <number>256</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="div_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="log_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>log</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="ln_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>ln</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="pushButton_delete">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="mul_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pow_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>^</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="mod_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>mod</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="tan_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>tan</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="pushButton_DEPOSIT">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>Депозитный</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QPushButton" name="pushButton_CREDIT">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>Кредитный</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="sqrt_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>sqrt</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="asin_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>asin</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="minus_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="plus_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="acos_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>acos</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="sin_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QPushButton" name="draw_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>График</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QPushButton" name="cos_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="atan_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>atan</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="compute_Btn">
       <property name="font">
        <font>
         <family>Noto Sans Oriya</family>
        </font>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
